src/target/xtensa/xtensa.h

```diff
@@ -31,13 +35,27 @@
  * Holds the interface to Xtensa cores.
  */
 
+/* Big-endian vs. little-endian detection */
+#define isbe(X)         ((X)->core_config->bigendian)
```

Comment:
By convention, It would be better to capitalize all macros.

Reply:
Good idea.  Changing to XT_ISBE().